srv/_modules/setup_conf.py

```diff
@@ -17,37 +17,60 @@
 
 # How to test:
 # salt-call saltutil.clear_cache && salt-call saltutil.sync_modules
-# salt-call setup_conf.conf_cmd "../files/samples/setup.yaml" 'test:post_install'
+# salt-call setup_conf.conf_cmd "../files/samples/setup.yaml"
+#   'test:post_install'
+
 import errno
+import logging
 import os
 import yaml
 
+
+logging.basicConfig(format='%(levelname)s:%(message)s')
+logger = logging.getLogger(__name__)
+
+
 def conf_cmd(conf_file, conf_key):
-  if not os.path.exists(conf_file):
-    print("[ERROR   ] Setup config file {0} doesn't exist.".format(conf_file))
-    raise FileNotFoundError(
-      errno.ENOENT,
-      os.strerror(errno.ENOENT),
-      conf_file
-    )
-
-  # print("Setup config file {0}".format(conf_file))
-
-  ret_val = ''
-  with open(conf_file, 'r') as fd:
-    try:
-      yml_dict = yaml.safe_load(fd)
-
-      script_path = yml_dict[conf_key.split(':')[0]][conf_key.split(':')[1]]['script']
-      if script_path and os.path.exists(script_path):
-        args = yml_dict[conf_key.split(':')[0]][conf_key.split(':')[1]]['args']
-        if isinstance(args, list):
-          args = ' '.join(args)
-
-        ret_val = script_path + " " + str(args)
-
-    except yaml.YAMLError as ymlerr:
-      print("Error parsing yaml file {0}".format(ymlerr))
-      ret_val = None
-  
-  return ret_val
+    if not os.path.exists(conf_file):
+        logger.error(f"Setup config file {conf_file} doesn't exist.")
+        raise FileNotFoundError(
+          errno.ENOENT,
+          os.strerror(errno.ENOENT),
+          conf_file
+        )
+
+    logger.debug(f"Setup config file {conf_file}")
+
+    replacement_url = __pillar__['provisioner']['common_config']['url']
+    ret_val = ''
+    with open(conf_file, 'r') as fd:
+        try:
+            yml_dict = yaml.safe_load(fd)
+
+            # This split is hard-coded as this is the input format expected
+            # during call from the sls file.
+            root_node = conf_key.split(':')[0]
+            config_stage = conf_key.split(':')[1]
+            cmd_path = yml_dict[root_node][config_stage]['cmd']
+            logger.debug(f"The command path from yaml file: {cmd_path}")
```

Comment:
Change debug message to something like this:
**_Component Setup Cmd: csm_setup init_**